Platon/backend/app/follow_system/views.py

```diff
@@ -1,10 +1,250 @@
+            except:
+                return make_response(jsonify({'error': 'Database Connection Error'}), 500)
+
+            if len(followSearch)==0:
+                return make_response(jsonify({'error': 'Follow Request not found'}), 404)
+
+            # If it exists: Create a Follow instance. Delete FollowRequest instance.
+            follow_request = followSearch[0]
+            new_follow_entry = Follow(follow_request.follower_id, follow_request.following_id)
+            try:
+                db.session.add(new_follow_entry)
+                db.session.delete(follow_request)
+                db.session.commit()
+            except:
+                return make_response(jsonify({'error': 'Database Connection Error'}), 500)
+
+            return make_response(jsonify({'msg': 'Follow Request is succesfully accepted'}), 200)
+
+        else:
+            return make_response(jsonify({'error': 'Input Format Error'}), 400)
+
+
+# login_required decorator will be added.
+class RejectFollowRequestAPI(Resource):
+
+    @api.doc(responses={200: 'Follow Request is rejected successfully',
+                        400: 'Input Format Error',
+                        401: 'Current user is unauthorized to reject the Follow Request',
+                        404: 'Follow Request not found',
+                        500: 'Database Connection Error'})
+    @api.expect(reject_follow_requests_parser)
+    @login_required
+    def post(user_id, self):
+        '''
+            Takes the follower_id and following_id as inputs and rejects the corresponding Follow Request.
+        '''
+        form = AcceptFollowRequestsForm(request.form)
+        if form.validate():
+
+            # Current user should be the following.
+            if user_id != form.following_id.data:
+                return make_response(jsonify({'error': 'Unauthorized'}), 401)
+
+            # Check if the given FollowRequest instance exists.
+            try:
+                followSearch = FollowRequests.query.filter(FollowRequests.following_id == form.following_id.data,
+                                                           FollowRequests.follower_id == form.follower_id.data).all()
+            except:
+                return make_response(jsonify({'error': 'Database Connection Error'}), 500)
+
+            if len(followSearch) == 0:
+                return make_response(jsonify({'error': 'Follow Request not found'}), 404)
+
+            # If it exists: Delete FollowRequest instance.
+            follow_request = followSearch[0]
+            try:
+                db.session.delete(follow_request)
+                db.session.commit()
+            except:
+                return make_response(jsonify({'error': 'Database Connection Problem'}), 500)
+
+            return make_response(jsonify({'msg': 'Follow Request is succesfully accepted'}), 200)
+
+        else:
+            return make_response(jsonify({'error': 'Bad Request'}), 400)
```

Comment:
I think these 2 classes can be combined. Because the answer to a follow request can only be positive or negative. So, you can combine these classes into a single endpoint by only adding another parameter that defines the state of the answer.

Also this can be a PUT request of the FollowRequestsAPI class. But I am not really sure about that. More opinion can be useful.

Reply:
> I think these 2 classes can be combined. Because the answer to a follow request can only be positive or negative. So, you can combine these classes into a single endpoint by only adding another parameter that defines the state of the answer.
> 
> Also this can be a DELETE request of the FollowRequestsAPI class. Because actually you delete the FollowRequest record from the database. But I am not really sure about that. More opinion can be useful.

As far as I've checked so far, the only missing update is the one regarding this part. @cakmadam98 has implemented all the other suggestions from @HUmutOzdemir. So, after the replacement of "AcceptFollowRequests" and "RejectFollowRequests" with single "FollowRequestsAPI", we can merge this code.

Reply:
With the latest commit, I've merged these 2 APIs into ReplyFollowRequestsAPI.